fedimint-core/src/lib.rs

```diff
@@ -135,6 +135,15 @@ pub enum BitcoinAmountOrAll {
     Amount(#[serde(with = "bitcoin30::amount::serde::as_sat")] bitcoin30::Amount),
 }
 
+impl std::fmt::Display for BitcoinAmountOrAll {
```

Comment:
Can we add a test (or expand the existing `test_deserialize_amount_or_all`) to ensure the round-trip conversion between `BitcoinAmountOrAll` and `String`?

Reply:
Expanded `test_deserialize_amount_or_all`